pontoon/sync/changeset.py

```diff
@@ -147,6 +147,12 @@ def update_entity_translations_from_vcs(
             )
         approved_translations = []
 
+        # Unfuzzy existing fuzzy translations to prevent duplicate fuzzies.
+        for translation in db_translations:
```

Comment:
I think it might cause more serious problems. A quick skim of django-bulk-update reveals that [they append all the pks of objects to be updated in a single list](https://github.com/aykut/django-bulk-update/blob/master/bulk_update/helper.py#L94), and eventually use that list in one big [generated update statement](https://github.com/aykut/django-bulk-update/blob/master/bulk_update/helper.py#L146). My expectation is that that would fail, but we'd need to test it to be sure.

> In old_translations we only keep approved ones.

You don't necessarily have to add them to `old_translations`, but you'll need to use that value when figuring out what translations have already been added to the list of translations to update.